docs/porting/overview.md

```diff
@@ -1,5 +1,70 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+If you've got code running on .NET Framework you may be interested in running your code on .NET Core.  This article will cover evaluating if porting to .NET Core is reasonable for you, an overview of the porting process, and a list of the tools you may use to port.
+
+## Should you port?
+
+The first thing you need to do is evaluate if it makes sense to port .NET Framework code over to .NET Core in the first place.  .NET Core offers some attractive capabilities, such as:
+
+- Using ASP.NET Core to write great web apps and services which you can write and deploy on your OS of choice
+- Console applications that can be Ahead-of-time compiled into native code with minimal dependencies
+- The ability to deploy apps and services app-local and with different versions of .NET Core side-by-side
```

Comment:
"app-local" is used a number of times in the topic, but it's not defined, and I doubt that all developers will know what it means. I one-sentence definition would be helpful.